netstandard/tools/NETStandard.Library.targets

```diff
@@ -0,0 +1,49 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(NETStandardToolsTaskDirectory)' == ''">
+    <NETStandardToolsTaskDirectory>$(MSBuildThisFileDirectory)</NETStandardToolsTaskDirectory>
+    <NETStandardToolsTaskDirectory Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)desktop/</NETStandardToolsTaskDirectory>
+  </PropertyGroup>
+
+  <Choose>
+    <When Condition="'$(ResolvePackageDependenciesForBuild)' == 'true' and '$(EnableResolvePackageDependencies)' == 'true' and Exists('$(ProjectAssetsFile)')">
```

Comment:
I copied this from https://github.com/dotnet/sdk/blob/d596f9e1d30ec5fa80cfb8534e1c94f17ae6f9db/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets#L118, though it looks like @natidea recently updated: https://github.com/dotnet/sdk/commit/057c7b650aae10764262fa6eb60875d5eec41c1b#diff-4a26943c6456c3afd827c7b87e69d390R117